src/routines.spec.ts

```diff
@@ -113,3 +117,20 @@ test(
     m.expect(error$).toBeObservable(errorMarbles, errors);
   })
 );
+
+test(
+  'tapRoutine register a routine',
+  marbles((m, t) => {
+    const action$ = m.hot(singleActionMarble, actions);
+    const action = actionCreator<{ letter: string }>('[Mock] action');
+    const routineToSubscribe = tapRoutine(action, (payload) =>
+      t.is(payload.letter, 'F')
```

Comment:
Since this is inside a payload, I think we should do `t.plan(1)` (docs on plan [here](https://github.com/avajs/ava/blob/main/docs/recipes/when-to-use-plan.md)).